src/directives/my_identity.ts

```diff
@@ -55,7 +55,8 @@ export default [
             }],
             template: `
                 <div class="my-threema-information" ng-click="ctrl.showQRCode()">
-                    <div class="nickname" ng-cloak ng-bind-html="ctrl.identity.publicNickname | emojify">
+                    <div class="nickname" ng-cloak
+                        ng-bind-html="ctrl.identity.publicNickname | emojify | emptyToPlaceholder: '-'">
```

Comment:
At first I thought this is an exploit (if the nickname contains a `<script>` tag), but Angular sanitizes input in ng-bind-html b default. :+1: